src/styles/utils/_variabless.scss

```diff
@@ -0,0 +1,15 @@
+$main-color: #e9967a;
+$bg-color: #000;
+$day-size: 100px;
+$padding-size: 24px;
+$width: ($day-size * 7) + ($padding-size * 2) + 6;
+$month-name: "July";
+$month-length: 31;

```

Comment:
Nice, but by task description you need to add month-length bem modifier to calendar class )

Reply:
Thanks a lot. I will fix it)